cpp/src/Ice/SSL/SchannelTransceiverI.cpp

```diff
@@ -457,39 +471,48 @@ Schannel::TransceiverI::sslHandshake()
     _writeBuffer.b.reset();
     _writeBuffer.i = _writeBuffer.b.begin();
 
+    if (_remoteCertificateValidationCallback &&
```

Comment:
This was previously done in "initialize", but moved here as we want to do this also when SSL handshake is called because of a renegotiation.